Packs/SafeNet_Trusted_Access/Scripts/STAFetchListContent/STAFetchListContent.py

```diff
@@ -0,0 +1,38 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import traceback
+
+''' STANDALONE FUNCTION '''
+
+
+# Get High Risk Group name from the list.
+def get_high_risk_group_sta(args: Dict[str, Any]) -> CommandResults:
+
+    list_name = args.get('list_name')
```

Comment:
If this is a script argument, please add to yml and readme.

Reply:
As discussed, no change needed.